### PR TITLE
add blogs section and release_202404

### PR DESCRIPTION
**Pull request by superstar54**
- Add a new `Blogs` section on the docs page. Then add the blog for the release of QEApp 2024.04 (news and details)

### CODE REVIEW

**Comment by superstar54** [comment]
Hi @unkcpz , I added the pseudopotential widget as you suggested, cheer!

**Comment by superstar54** [comment]
Hi @AndresOrtegaGuerrero , as you suggested, I moved the "total magnetization" from `bug fixes` to `new feature`.

File: docs/source/blogs/release_202404.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. _blogs:release_202404:
+
+************************
+QEApp 2024.04 Release
+************************
+
+AiiDAlab team and the online developers community are proud to present QEApp 2024.04!
+
+This version represents a significant milestone in the evolution of QEApp, delivering numerous features and improvements across various aspects of the application.
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:9]:
```suggestion
The AiiDAlab team and the broader developer community is proud to present the AiiDAlab Quantum Espresso app (QEApp) version 2024.04!

This version represents a significant milestone in the evolution of QEApp, delivering several features and improvements on various aspects of the app.
```

File: docs/source/blogs/release_202404.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+What's New
+================================
+
+New Plugins
+----------------------
+We are introducing a range of new plugins that expand the capabilities of QEApp.
+These plugins cater to diverse computational needs and enhance the overall user experience.
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:17]:
```suggestion
After the introduction of a plugin interface in the previous release of QEApp, we are now introducing several new plugins that expand the capabilities of QEApp.
These plugins focus on addressing several diverse computational needs, significantly enhancing the range of properties that are now offered to the QEApp users.
```

File: docs/source/blogs/release_202404.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+.. figure:: /_static/images/qeapp_release_202404_plugins.png
+   :align: center
+   :alt: New plugins for QEApp 2024.04
+
+Plugin Management Page
+----------------------
+The new plugin management page simplifies how users find and manage their plugins, making it easier to customize and enhance their computational environment.
+
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:26]:
```suggestion
The new plugin management page simplifies how users can find and manage their plugins, making it easier to customize and enhance their computational environment.

```

File: docs/source/blogs/release_202404.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+.. figure:: /_static/images/qeapp_release_202404_plugin_management.gif
+   :align: center
+   :alt: Plugin management interface
+
+Job Search Page
+----------------------
+To facilitate better job management, we have added a new job search page that allows users to efficiently search and manage their computational jobs.
+
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:34]:
```suggestion
To facilitate better job management, we now provide a new job search page that allows users to efficiently search and manage their computational jobs run by the QEApp.

```

File: docs/source/blogs/release_202404.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+.. figure:: /_static/images/qeapp_release_202404_job_list.gif
+   :align: center
+   :alt: New job management page
+
+New Widgets
+----------------------
+This release also introduces several new widgets, including tools for DFT+U, magnetic settings, computational resources and pseudopotential setup, further enhancing the functionality and user flexibility of QEApp.
+
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:42]:
```suggestion
This release also introduces several new widgets, including tools for dealing with DFT+U calculations, with magnetic settings, with setting up custom computational resources and selecting custom pseudopotentials, and more generally further enhancing the functionality and user interface of the QEApp.

```

File: docs/source/blogs/release_202404.rst [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+   :align: center
+   :alt: New widgets in QEApp
+
+For a deeper dive into all the updates, please visit the :ref:`blogs:release_202404_details` for an in-depth look at what's new.
+
+Acknowledgments
+================================
+We extend our heartfelt thanks to all contributors from the AiiDAlab community and our supporting organizations. Your dedication and contributions make these advancements possible.
+
+Happy calculation!
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:53]:
```suggestion
We extend our heartfelt thanks to all contributors from the AiiDAlab community, to the AiiDA developers, and to our supporting organizations. Your dedication and contributions make these advancements possible.

Happy calculations!
```

File: docs/source/blogs/release_202404.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+The AiiDAlab Team
+April 25th, 2024
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404.rst:56]: I guess we need to update the date. Shall we call the release 2024.05 at this point? Or we keep the name? (OK to keep the name if complex to change, and we release in a few days, in my opinion)

File: docs/source/blogs/release_202404.rst (hunk 6 above)
Comment (giovannipizzi) [reply docs/source/blogs/release_202404.rst:53]: Maybe "supporting organisations" should be a link to the section "Our sponsors" of https://www.aiidalab.net/about/

File: docs/source/blogs/release_202404_details.rst [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+- Support using customize pseudopotential by @unkcpz in `#435 <https://github.com/aiidalab/aiidalab-qe/pull/435>`_
+
+New registered plugins:
+
+- `aiida-bader <https://github.com/superstar54/aiida-bader>`_ by @superstar54
+- `aiidalab-qe-muon <https://github.com/mikibonacci/aiidalab-qe-muon>`_ by @mikibonacci
+- `aiidalab-qe-vibroscopy <https://github.com/mikibonacci/aiidalab-qe-vibroscopy>`_ by @mikibonacci and @AndresOrtegaGuerrero
+
+
+User friendly improvements:
```
Comment (giovannipizzi) [inline docs/source/blogs/release_202404_details.rst:29]:
```suggestion
User-friendliness improvements:
```

**Review by giovannipizzi — COMMENTED** [review]
Hi @superstar54, I did some suggestions on the text, feel free to further edit if needed! But then it can go, for me! Thanks!

File: docs/source/blogs/release_202404.rst [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+   :align: center
+   :alt: New job management page
+
+New Widgets
+----------------------
+This release also introduces several new widgets, including tools for dealing with DFT+U calculations, with magnetic settings, with setting up custom computational resources and selecting custom pseudopotentials, and more generally further enhancing the functionality and user interface of the QEApp.
+
+.. figure:: /_static/images/qeapp_release_202404_new_widgets.png
+   :align: center
+   :alt: New widgets in QEApp
```
Comment (AndresOrtegaGuerrero) [inline docs/source/blogs/release_202404.rst:45]: New widgets in AiiDAlab QEApp

File: docs/source/blogs/release_202404.rst [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+================================
+
+New Plugins
+----------------------
+After the introduction of a plugin interface in the previous release of QEApp, we are now introducing several new plugins that expand the capabilities of QEApp.
+These plugins focus on addressing several diverse computational needs, significantly enhancing the range of properties that are now offered to the QEApp users.
+
+.. figure:: /_static/images/qeapp_release_202404_plugins.png
+   :align: center
+   :alt: New plugins for QEApp 2024.04
```
Comment (AndresOrtegaGuerrero) [inline docs/source/blogs/release_202404.rst:21]: New plugins for AiiDAlab QEApp 2024.04

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM!  , Nice work, thank you!

File: docs/source/blogs/release_202404.rst (hunk 9 above)
Comment (superstar54) [reply docs/source/blogs/release_202404.rst]: Done!

File: docs/source/blogs/release_202404.rst (hunk 10 above)
Comment (superstar54) [reply docs/source/blogs/release_202404.rst]: Done!

File: docs/source/blogs/release_202404_details.rst [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+******************************
+
+New Feature:
+
+- Support using total magnetization by @AndresOrtegaGuerrero in `#512 <https://github.com/aiidalab/aiidalab-qe/pull/512>`_
+- Add plugin: XPS by @superstar54 in `#518 <https://github.com/aiidalab/aiidalab-qe/pull/518>`_
+- Add new computational resource widget by @superstar54 in `#566 <https://github.com/aiidalab/aiidalab-qe/pull/566>`_
+- Support DFT+U by @AndresOrtegaGuerrero in `#577 <https://github.com/aiidalab/aiidalab-qe/pull/577>`_
+- Add plugin: XAS Plugin by @PNOGillespie in `#580 <https://github.com/aiidalab/aiidalab-qe/pull/580>`_
+- New BandsPdosWidget by @AndresOrtegaGuerrero in `#581 <https://github.com/aiidalab/aiidalab-qe/pull/581>`_
```
[the remark this answers is not in the document]
Comment (superstar54) [reply docs/source/blogs/release_202404_details.rst:14]: Done!

File: docs/source/blogs/release_202404.rst (hunk 7 above)
Comment (superstar54) [reply docs/source/blogs/release_202404.rst]: Thanks for the review! I have updated the date!
We kept the name 2024.04 even though there were a few days of delay.

**Pull request merged**